<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tragaci</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="list_widget_finders"/>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_load_data">
       <property name="text">
        <string>Ucitaj podatke</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mapa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="table_wdiget_map"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Brzina igre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="spin_box_game_speed">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="push_button_start_game">
       <property name="text">
        <string>Zapocni igru</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
